stl/inc/xutility

```diff
@@ -5148,8 +5132,29 @@ _INLINE_VAR constexpr bool _Memchr_in_find_is_safe =
             > && !is_volatile_v<remove_reference_t<_Iter_ref_t<_Iter>>>;
 
 template <class _InIt, class _Ty>
-_NODISCARD constexpr _InIt _Find_unchecked1(_InIt _First, const _InIt _Last, const _Ty& _Val, false_type) {
-    // find first matching _Val
+_NODISCARD _CONSTEXPR20 _InIt _Find_unchecked(_InIt _First, const _InIt _Last, const _Ty& _Val) {
+    // find first matching _Val; choose optimization
+    // activate optimization for contiguous iterators to (const) bytes and integral values
+    // find first byte matching integral _Val

```

Comment:
This comment should move immediately within `if constexpr (_Memchr_in_find_is_safe<_InIt, _Ty>)` because that's where we determine that we're working with a range of bytes and integral `_Val`. (The comment was originally within the `true_type` overload.)